loglist2/loglist2_test.go

```diff
@@ -39,6 +39,15 @@ func mustParseUnixTime(sTime string) time.Time {
 	return tm.UTC()
 }
 
+// mustParseRFC3339Time is helper func
+func mustParseRFC3339Time(sTime string) time.Time {
```

Comment:
Why not just have a single `mustParseTime()` func that takes a format string? Then you could do either `mustParseTime(time.UnixDate, "...")` or `mustParseTime(time.RFC3339, "...")`.

Reply:
Done